addon/components/lt-body.js

```diff
@@ -239,6 +292,57 @@ export default Component.extend({
     this.set('useVirtualScrollbar', fixedHeader || fixedFooter);
   },
 
+  didReceiveAttrs() {
```

Comment:
Would it be better to handle this with `didUpdateAttrs` since it gives you the before and after values? That could remove the need of having `_scrollTo` and `_scrollToRow`. You can handle the initial scroll with `didInsertElement`